.github/workflows/build.yml

```diff
@@ -49,9 +54,33 @@ jobs:
           free -m
           docker system df
           docker system df -v
-      - name: Build EVE KVM
+
+  eve:
```

Comment:
I can see the wrong order of builds, seems we should add `needs: packages` for `eve` job.

Reply:
Correct. Fixed.